exec/cnex/global.c

```diff
@@ -130,10 +131,17 @@ TDispatch gfuncDispatch[] = {
     PDFUNC("runtime$executorName",      runtime_executorName),
     PDFUNC("runtime$moduleIsMain",      runtime_moduleIsMain),
 
-    // System - System level calls
+    // string - string module functions
+    PDFUNC("string$find",               string_find),
+    PDFUNC("string$lower",              string_lower),
+    PDFUNC("string$split",              string_split),
+    PDFUNC("string$splitLines",         string_splitLines),
+    PDFUNC("string$trimCharacters",     string_trimCharacters),
+    PDFUNC("string$upper",              string_upper),
+
+    // sys - System level calls
     PDFUNC("sys$exit",                  sys_exit),
 
-    // Textio - Text file modules
```

Comment:
What happened to this line? You've got heading comments for all the other sections.

Reply:
Humph.  That's a new one!  Ok, added that comment back in.